src/wallet/rpc/transactions.cpp

```diff
@@ -444,7 +444,10 @@ RPCHelpMan listtransactions()
     return RPCHelpMan{
         "listtransactions",
         "If a label name is provided, this will return only incoming transactions paying to addresses with the specified label.\n"
-                "\nReturns up to 'count' most recent transactions skipping the first 'from' transactions.\n",
+                "Returns up to 'count' most recent transactions ordered from oldest to newest while skipping the first 'from' transactions.\n"
```

Comment:
q: how the user would understand what 'from' is on this message?
As far as I can see, the message is referring to the "skip" arg instead.

Reply:
> the message is referring to the "skip" arg instead.

I guess yeah, maybe there was a "from" argument earlier here?
I somehow automatically ignored this as I was focussed on adding the example below, but I will push a commit rewording this sentence.